inference-engine/tests/ngraph_functions/src/activation.cpp

```diff
@@ -28,6 +28,11 @@ std::shared_ptr<ngraph::Node> makeActivation(const ngraph::Output<Node> &in,
     auto hard_sigmoid_beta = std::make_shared<ngraph::op::Constant>(
             type, ngraph::Shape(), 0.5f);
 
+    auto negative_slope = std::make_shared<ngraph::op::Constant>(
+            ngraph::element::f32,
+            ngraph::Shape{1},
+            std::vector<float>{-0.01f});
```

Comment:
@iefode We have [the issue](https://jira.devtools.intel.com/browse/CVS-32032) which says that CPU can't handle the case when negative_slope input is not constant. As I can see your test doesn't cover this case.

Also in most models negative_slope has per-channels values. From my perspective it is worthwhile to cover both situations: per-tensor and per-channel.

Reply:
Added, please review